rcl/include/rcl/logging_external_interface.h

```diff
@@ -31,7 +31,7 @@
 RCL_PUBLIC
 RCL_WARN_UNUSED
 rcl_ret_t
-rcl_logging_external_initialize(const char * config_file);
+rcl_logging_external_initialize(const char * config_file, rcutils_allocator_t allocator);
```

Comment:
We discussed offline about the fact that this needs to be deduplicated, but @clalancette is just going to make a TODO for now.